vector-app/src/debug/java/im/vector/app/flipper/VectorFlipperProxy.kt

```diff
@@ -44,6 +45,11 @@ class VectorFlipperProxy @Inject constructor(
     override fun init(matrix: Matrix) {
         SoLoader.init(context, false)
 
+        // https://github.com/facebook/flipper/issues/3572
+        if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.LOLLIPOP_MR1) {
+            return
+        }
```

Comment:
Why this is not done above the line `SoLoader.init(context, false)`?

Also I would also update the fun `networkInterceptor()` to return null if `Build.VERSION.SDK_INT <= Build.VERSION_CODES.LOLLIPOP_MR1`

Reply:
I put it here to keep this logic at the same point as the existing check (on the line immediately after this one).

[In the docs `SoLoader.init()` is always called](https://fbflipper.com/docs/getting-started/android-native/#application-setup) regardless of whether Flipper is enabled but I'm not sure if there's any reason for that.

The same is true of the `networkInterceptor()` but I'd agree, it should return null if Flipper is disabled.

Reply:
[`5425622`](https://github.com/vector-im/element-android/pull/7428/commits/542562216d1a45a67d1bf68f9a944b96a6e3cb7a)